platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -494,6 +494,11 @@ bool OS_LinuxBSD::_check_internal_feature_support(const String &p_feature) {
 		return true;
 	}
 
+	// Match against the specific OS (linux, freebsd, etc).
+	if (p_feature == get_name().to_lower()) {
+		return true;
+	}
```

Comment:
Shouldn't this be `get_identifier` to match `OS::has_feature`? The contents of the method are `return get_name().to_lower();`. But then... how is this code any different from this code already in `OS::has_feature`?

	if (p_feature == get_identifier()) {
		return true;
	}

Reply:
This is the code for `get_name()`:

```
String OS_LinuxBSD::get_name() const {
#ifdef __linux__
	return "Linux";
#elif defined(__FreeBSD__)
	return "FreeBSD";
#elif defined(__NetBSD__)
	return "NetBSD";
#elif defined(__OpenBSD__)
	return "OpenBSD";
#else
	return "BSD";
#endif
}
```

It's different depending on the OS, and gets 'linux' working on Linux again. `get_identifier()` is just this:

```
String OS_LinuxBSD::get_identifier() const {
	return "linuxbsd";
}
```

Reply:
Ah, I see. Why do we have `OS_LinuxBSD::get_identifier()` separately overwritten if it also has a default implementation in the OS class?

It looks like the only other place that calls this method is `RemoteFilesystemClient::_synchronize_with_server`, which I am suspicious of, do we really want to have that class use the tag `linuxbsd` instead of `linux` and *`bsd` separately?

Reply:
> It looks like the only other place that calls this method is RemoteFilesystemClient::_synchronize_with_server, which I am suspicious of, do we really want to have that class use the tag linuxbsd instead of linux and *bsd separately?

I believe this was the intentional change: the PR that caused the regression (https://github.com/godotengine/godot/pull/76540) is about reworking how the remote file system works. I think causing the 'linux' and BSD feature tags to stop working was an unintended consequence of that.